students/Z_shen/lesson09/donor_models.py

```diff
@@ -0,0 +1,68 @@
+class Donor(object):
+    def __init__(self, name=None, amount=None):
+        self.name = name.title()
+        self.amount = []
+        if amount is not None:
+            self.amount = amount
+
+    def __repr__(self):
+        return '{} donated ${}'.format(self.name, self.amount)
+
+    def __str__(self):
+        return 'Donor({}, {})'.format(self.name, self.amount)
+
+    def send_letter(self):
+        content = ('''Dear {}, 
+            Thank you for your generous donation of ${:,.2f} to us.
+            It will be put to very good use.
+
+                                Sincerely,
+                                    -The Team
+                                              '''.format(self.name, self.amount))
+        return content
+
+
+class DonorCollection:
+
+    def __init__(self, donor_list=None):
+        self.donors = {}
+        if donor_list is not None:
+            self.donors = donor_list
```

Comment:
you should be using your Donor class here

create Donor instance and store it in this collection class, here is some stubbed out code as an example:
https://repl.it/@naaleksandrova/Lesson9Example